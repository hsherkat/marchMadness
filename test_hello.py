from approvaltests.approvals import verify


def test_simple():
    result = """
    Hello world!
    This is an approval test!
    """
    verify(result)
