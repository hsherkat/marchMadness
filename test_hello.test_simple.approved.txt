
    Hello world!
    This is an approval test!
    
